SECP-service/src/main/resources/assets/app/directives/secp-manage-modal/secp-manage-modal.html

```diff
@@ -15,13 +15,22 @@ <h4 class="modal-title">{{title}}</h4>
                            required="required"><br><br>
                 </div>
 
-                Permissions:<br>
-                <select select2 required class="permission-level-select" style="width: 100%" ng-model="modalData.permissions"
-                        ng-options="permission.name for permission in permissions track by permission.id" multiple>
-                </select>
+                <div ng-if="!isuser">
+                    Permissions:<br>
+                    <select select2 class="permission-level-select" style="width: 100%" ng-model="modalData.permissions"
+                            ng-options="permission.name for permission in permissions track by permission.id" multiple>
+                    </select>
+                </div>
+
+                <div ng-if="isuser">
+                    Permission:<br>
+                    <select select2 required class="permission-level-select" style="width: 100%" ng-model="modalData.permission"
+                            ng-options="permission.name for permission in permissions track by permission.id">
+                    </select>
+                </div>
 
                 <br><br>Roles:<br>
```

Comment:
Do we need to check isuser for roles?

Reply:
no the reason why i put it for permission is because a user CAN only have one permission. so no need of the multiple keyword. For roles a user could have many. same goes for groups.

Reply:
Perfect